Gruntfile.coffee

```diff
@@ -1141,17 +1141,12 @@ module.exports = (grunt) ->
 					expand: true
 					flatten: true
 				,
-					cwd: "node_modules/mathjax"
+					cwd: "node_modules/mathjax/es5"
 					src: [
-						"MathJax.js"
-						"config/**"
-						"extensions/**"
-						"jax/**"
-						"localization/**"
-						"fonts/HTML-CSS/TeX/woff/**"
-						"fonts/HTML-CSS/TeX/otf/**"
+						"mml-chtml.js"
+						"output/chtml/fonts/woff-v2/**"
 					]
-					dest: "<%= coreDist %>/js/MathJax/"
+					dest: "<%= coreDist %>/js/MathJax/es5/"
```

Comment:
just put in the root of "MathJax" folder

Reply:
Done.